events_webhook/src/main/java/com/adobe/aio/event/webhook/cache/CacheService.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.aio.event.webhook.cache;
+
+import com.adobe.aio.event.webhook.model.CacheableObject;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public interface CacheService {
```

Comment:
I would really like to get rid of this cache.
our customer can easily use any cache service/framework they want.
this PR should only be about showing how to validate the message headers' signature and no more